src/pages/kyunghyun/Login/Login.scss

```
@@ -0,0 +1,200 @@
+@mixin flex-column {
+  display: flex;
+  flex-direction: column;
+}
+@mixin flex-center {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+@mixin out-box {
+  background-color: #fff;
+  border: 1px solid #dbdbdb;
+  border-radius: 3px;
+}
+
+@mixin ling-font {
+  color: #385185;
+  text-align: center;
+  text-decoration: none;
+}
+
+@mixin link-text {
+  text-decoration: none;
+  text-align: center;
+}
+
+@mixin blue-text {
+  color: #385185;
+  @include link-text;
+}
+
+.KyungHyunLogin {
+  transform: translate(-50%, -50%);
+  top: 50%;
+  left: 50%;
+  position: absolute;
+  align-items: center;
+  width: 372px;
+  height: 561px;
+
+  header {
+    @include out-box;
+    @include flex-column;
+    align-items: center;
+    width: 100%;
+    padding: 10px 0;
+    margin-bottom: 10px;
+    h1 {
+      display: block;
+      width: auto;
+      margin: 22px 12px;
+      height: 51px;
+      font-family: 'Style Script', cursive;
+      text-align: center;
+      color: #000;
+      font-size: 50px;
+      font-weight: 800;
+    }
+    .main-function {
+      @include flex-column;
+      @include flex-center;
+      align-items: center;
+      width: 360px;
+      height: 263px;
+      margin-bottom: 10px;
+      .login-form {
+        @include flex-column;
+        width: 100%;
+        .id,
```

```suggestion
      margin-bottom: 10px;
      
      .login-form {
        @include flex-column;
        width: 100%;
        
        .id,
```
새로운 셀렉터 앞에서는 한줄 띄워주세요